myria3d/pctl/dataset/utils.py

```diff
@@ -171,16 +162,16 @@ def make_circle_wkt(center, subtile_width):
     return wkt
 
 
-def get_las_paths_by_split_dict(data_dir: str, split_csv_path: str) -> LAS_PATHS_BY_SPLIT_DICT_TYPE:
+def get_las_paths_by_split_dict(
+    data_dir: str, split_csv_path: str
+) -> LAS_PATHS_BY_SPLIT_DICT_TYPE:
     las_paths_by_split_dict: LAS_PATHS_BY_SPLIT_DICT_TYPE = {}
     split_df = pd.read_csv(split_csv_path)
     for phase in ["train", "val", "test"]:
-        basenames = split_df[
-            split_df.split == phase
-        ].basename.tolist()
-        las_paths_by_split_dict[phase] = [
-            find_file_in_dir(data_dir, b) for b in basenames
-        ]
+        basenames = split_df[split_df.split == phase].basename.tolist()
+        # Explicit data structure with ./val, ./train, ./test subfolder is required.
+        # TODO: indicate this in the doc as well.
```

Comment:
is it still a TODO or is [this text](https://github.com/IGNF/myria3d/pull/81/files#diff-5a97c8b09e9bd0f81c026127e37959916b939c5eb258abe75dcc871bd865c24aR24) what you wanted to add to the doc?

Reply:
This is not a todo anymore and the link is correct ; will remove the comment !

Reply:
Done